buildpack/telemetry/appdynamics.py

```diff
@@ -18,6 +18,40 @@
     "machine-agent",
 )
 
+CF_APPLICATION_INDEX = os.getenv("CF_INSTANCE_INDEX", default=0)
+CF_APPLICATION_NAME = util.get_vcap_data()["application_name"]
```

Comment:
This gives you the environment ID, which is not necessarily interesting to customers. Is this by design?

Reply:
@markbastiaans The environment ID is provided as a "tier name". From the AppDynamics perspective, the "application name" is application name from domain, and that is the same as on Developer Portal, which makes sense for a user. Also, we consider "environment ID" as a logical value for "tier name" since it reflects the hierarchy on Developer portal. Also  user can find environment ID on Developer Portal.

Example hierarchy: "test application" / <environment id> / node_0

Reply:
<img width="894" alt="Screenshot 2022-05-05 at 14 38 24" src="https://user-images.githubusercontent.com/74789092/166924797-d6ccec10-2e29-4b30-b817-7c15191c94d6.png">